### ISSUE TITLE
integration into odot release

### ISSUE THREAD

**Issue by ramagottfried** [CNMAT/CNMAT-ODOT-SS#18]
hey @equilet (and @kulpajj )

Ed suggested including the odot-ss in the main odot release as a more fleshed out odot tutorial.  

Just taking a quick peek, things look pretty good, but it could use some more instruction or explanation I think.  Probably you did the explaining verbally during the course, so maybe you could add a few notes just to give a little more guidance for new users?

any thoughts about the format?

For the Package Manager, I think we're going to combine odot and the Externals (that way it's fewer packages to download, and also the analyzer~ help patch has odot in it).

**Comment by equilet** [comment]
This is a good idea for a future release.  The ask is not contingent on us getting an ODOT release out there, though.  I'm closing this for now, with the understanding that both video and patch tutorials are a forthcoming aspect of ODOT work and should be taken up in that repository.  This repo is parked currently as a development space for my upcoming semester-length course at UCB, and will be made private for now.
